src/common/simulation.ts

```diff
@@ -255,20 +263,17 @@ function createWorld(map: GameMap): World {
     restitution: 0.0,
   };
 
-  const tileWidth = constants.TILE_WIDTH;
-  const tileHeight = constants.TILE_HEIGHT;
-  const halfWidth = tileWidth / 2;
-  const halfHeight = tileHeight / 2;
+  const halfWidth = 0.5 * constants.TILE_LOGICAL_SIZE;
 
   for (let y = 0; y < map.height; ++y) {
     for (let x = 0; x < map.width; ++x) {
       if (!map.at(x, y).walkable) {
         // TODO: ändra tillbaka till 1x1-rutor typ
         const center = Vec2(
-          x * tileWidth + halfWidth,
-          y * tileHeight + halfHeight,
+          x * constants.TILE_LOGICAL_SIZE + halfWidth,
+          y * constants.TILE_LOGICAL_SIZE + halfWidth,
         );
-        const shape: any = new Box(halfWidth, halfHeight, Vec2.zero(), 0.0);
+        const shape: any = new Box(halfWidth, halfWidth, Vec2.zero(), 0.0);
```

Comment:
Om bredd och höjd måste vara samma så borde det dokumenteras.

Reply:
vi kan väl dokumentera i våra huvuden?

Reply:
Det borde läggas in en kontroll när vi skapar ett `Tileset` då, och ändra till `halfSize` och byt våra bredder och höjder i `Tileset` till att bara ha en `size`.

Reply:
tilesetet kan ju fortfarande supporta olika storlekar, bara att vi bara ger dem kvadratiska

Reply:
Om vi aldrig kommer ha det och det inte ens kommer fungera så borde det ändras

Reply:
men whatever, kör

Reply:
var inte ens så jobbigt som jag trodde att det skulle vara https://github.com/erikvader/vov/pull/97/commits/16b96989b62f93c5bd7b12dbd7a21965a72a636f